core/lib/multivm/src/vm_instance.rs

```diff
@@ -263,8 +265,8 @@ impl<S: ReadStorage, H: HistoryMode> VmInstance<S, H> {
             VmVersion::Vm1_5_0IncreasedBootloaderMemory => match mode {
                 FastVmMode::Old => Self::new(l1_batch_env, system_env, storage_view),
                 FastVmMode::New => {
```

Comment:
Could we add a new mode for our vm instead of removing the `vm_fast`? If it is too complicated we can leave it for another pr though.

Reply:
It should not be difficult, I'll add it

Reply:
At the end it was not as easy as intended, there a two options for this, one is adding a new value under `core/lib/basic_types/src/vm.rs` `FastVmMode` , however we are not actually a new version of the fast vm, so I'm not sure this is the correct approach, also this would also mean a change in the `zksync-protobuf-config` repository.

Another option is to add our vm under `core/lib/basic_types/src/vm.rs` `VmVersion`, but that struct is directly related to `core/lib/basic_types/src/protocol_version.rs` `ProtocolVersionId`, and it does not seem right to add a new protocol version for our vm.

Reply:
Yeah, let's not worry about that for now.